apps/subscription-manager/src/subscription_manager.py

```diff
@@ -1,16 +1,24 @@
 import json
 import logging
 import os
+import sys
 
 import boto3
 
 import dynamo
 
+LAMBDA_CLIENT = boto3.client('lambda')
```

Comment:
I also prefer to see global variables initialized before any function definitions, but should this be after the logging setup so that any exceptions it throws are logged?

Reply:
We assign a lambda to excepthook instead of a function, but I don't know that it improves things much:
`sys.excepthook = lambda exc_type, value, traceback: logger.critical('Unhandled exception', exc_info=(exc_type, value, traceback))`

Reply:
Approved as-is, I haven't convinced myself we can do much better, although I'm open to thoughts on the matter.

Reply:
Ah good point, the logging setup should probably come before anything else. Another idea I had was to move the logging boilerplate:

```python3
logging.basicConfig()
logger = logging.getLogger()
logger.setLevel(logging.INFO)


def handle_exception(exc_type, value, traceback) -> None:
    logger.critical('Unhandled exception', exc_info=(exc_type, value, traceback))


sys.excepthook = handle_exception
```

somewhere else that could be reused across all Lambda functions, so we wouldn't have to copy-paste it for every Lambda. Would it work to just move it into a separate module and then `import` that module from every Lambda? I could experiment with that solution in `tools-sandbox` if we're not sure.

Reply:
I went ahead and merged because I want to start logging unhandled exceptions properly, but created https://github.com/ASFHyP3/hyp3/issues/1295 to track the logging boilerplate issue.